### ISSUE TITLE
Error on admin\models\translate.php error on ligne 194 - correction inside

### ISSUE THREAD

**Issue by florian-denizot** [JoomFish/joomfish-2.5#138]
should be:

$query->from($db->quoteName('#__'.$table->Name).' AS c');

and not 

$query->from($db->quoteName('#__'.$tableclass).' AS c');

cheers

**Comment by akempkens** [comment]
Can you give me an idea how you found this issue?

**Comment by akempkens** [comment]
https://github.com/JoomFish/joomfish-2.5/commit/e622b033b1b2363149e28e819fdecb3feae118ac

**Comment by florian-denizot** [comment]
I had a full bilingual site already translated with the core Joomla translation. When I installed Joomfish I needed to link every translated article, module and menu to it's original equivalent using the plugin that extend the core elements. 

From the translated element edit page when I was opening the modal window to choose the original element it was linked to, I was getting an error stating that the table, let say #__Menu - instead of #__menu - couldn't be found.

You probably didn't get any problem during your checks because by default, on Windows installation of Mysql, requests on table name are non-sensitive to the case unlike on Linux / Unix installation.
